creator-node/scripts/updateContentBlacklist.js

```diff
@@ -1,128 +1,292 @@
+    throw new Error(`Error with adding type [${type}] and id [${ids}] to ContentBlacklist: ${e}`)
   }
 
   return resp.data
 }
 
 /**
- * 1. Signs the data with PRIVATE_KEY specified in this script
+ * 1. Signs the data {type, id, timestamp} with PRIVATE_KEY specified in this script
  * 2. Sends axios request to remove entry from content blacklist of type and id
  * @param {string} type
- * @param {int} id
+ * @param {number[]} ids
+ */
+async function removeIdsFromContentBlacklist (type, ids) {
+  const { timestamp, signature } = generateTimestampAndSignature({ type, ids }, PRIVATE_KEY)
+
+  let resp
+  try {
+    resp = await axios({
+      url: `${CREATOR_NODE_ENDPOINT}/blacklist/remove/ids`,
+      method: 'post',
+      params: { type, ids, timestamp, signature },
+      responseType: 'json'
+    })
+  } catch (e) {
+    throw new Error(`Error with removing type [${type}] and ids [${ids}] from ContentBlacklist: ${e}`)
+  }
+
+  return resp.data
+}
+
+/**
+ * 1. Signs the data {cids, timestamp} with PRIVATE_KEY specified in this script
+ * 2. Sends axios request to add cids to content blacklist
+ * @param {string[]} cids
  */
-async function removeFromContentBlacklist (type, id) {
-  const { timestamp, signature } = generateTimestampAndSignature({ type, id }, PRIVATE_KEY)
+async function addCIDsToContentBlacklist (cids) {
+  const { timestamp, signature } = generateTimestampAndSignature({ cids }, PRIVATE_KEY)
 
   let resp
   try {
     resp = await axios({
-      url: `${CREATOR_NODE_ENDPOINT}/blacklist/delete`,
+      url: `${CREATOR_NODE_ENDPOINT}/blacklist/add/cids`,
       method: 'post',
-      params: { type, id, timestamp, signature },
+      params: { cids, timestamp, signature },
       responseType: 'json'
     })
   } catch (e) {
-    throw new Error(`Error with removing type [${type}] and id [${id}] from ContentBlacklist: ${e}`)
+    throw new Error(`Error with adding cids [${cids.toString()}] from ContentBlacklist: ${e}`)
   }
 
   return resp.data
 }
 
+/**
+ * 1. Signs the data {cids, timestamp} with PRIVATE_KEY specified in this script
+ * 2. Sends axios request to remove cids from content blacklist
+ * @param {string[]} cids
+ */
+async function removeCIDsFromContentBlacklist (cids) {
+  const { timestamp, signature } = generateTimestampAndSignature({ cids }, PRIVATE_KEY)
+
+  let resp
+  try {
+    resp = await axios({
+      url: `${CREATOR_NODE_ENDPOINT}/blacklist/remove/cids`,
+      method: 'post',
+      params: { cids, timestamp, signature },
+      responseType: 'json'
+    })
+  } catch (e) {
+    throw new Error(`Error with removing cids [${cids.toString()}] from ContentBlacklist: ${e}`)
+  }
+
+  return resp.data
+}
+
+/**
+ * 1. Get all blacklisted content
+ * 2. Iterate through passed in CLI args against fetched content
+ * @param {string} type
+ * @param {(number[]|string[])} list cids or ids
+ */
+async function verify (type, list) {
```

Comment:
i think the ideal verify test would actually be to fetch the track metadata and try fetching a segment from the cnode and make sure it's blacklisted. 

see my comment above about possibly fetching from dp